WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -688,6 +726,16 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     return nil;
 }
 
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath {
+    NSInteger settingsSection = [self.tableSections[indexPath.section] integerValue];
+    switch (settingsSection) {
+        case SiteSettingsSectionMedia:
+            return WPTableViewDefaultRowHeight * 1.5;
```

Comment:
Can we add a constant for this `1.5`, just to help explain why it's there? `MediaQuotaCellHeightMultiplier` or something?

Reply:
Actually, I just noticed that `MediaQuotaCell` has a `height` property... should we return that instead? It's set to 44, but that's not what we're using in this method.

Reply:
Good catch!